src/containers/ap/ap-registration/ApRequiredInfo.tsx

```diff
@@ -217,21 +217,6 @@ export default class ApRequiredInfo extends React.Component<Props, State> {
 					options={countries}
 					nullDisplay="- Select -"
 				/>
-				<PhoneTriBox<Form,  PhoneTriBoxProps<Form>>
```

Comment:
Did you cut this when you meant to copy?

Reply:
good catch, bad Evan

EM

Reply:
It's not diffing cleanly, not the end of the world but the easiest way to make it be exactly the way it was before is to run `git show release/facta:path/to/file > path/to/file`.  Same thing happened to package-lock which I already fixed.  Give it a try on this one